src/Runner.php

```diff
@@ -133,7 +133,8 @@ public function run($input = null, $output = null, $app = null, $commandFiles =
 
         // If we were not provided a container, then create one
         if (!$this->getContainer()) {
-            $container = Robo::createDefaultContainer($input, $output, $app);
+            $config = Robo::createConfiguration(['robo.yml']);
```

Comment:
As proposed, by default Robo always (and only) loads robo.yml from the cwd. Additional configuration files may be loaded from a RoboFile's constructor. Currently, there is no provision to skip loading robo.yml; however, the RoboFile could choose to re-import the new configuration, erasing whatever was in robo.yml if desired.